docs/src/troubleshooting.md

```diff
@@ -1,10 +1,18 @@
 ## Self-help instructions
 
-Enable LSP logging: In LSP Settings enable the `log_debug` setting.
-Enable server logging: set `log_server` to ["panel"]
-Run "LSP: Toggle Log Panel" from the command palette. No restart is needed.
+To get more visibility into the inner-workings of the LSP client and the server and be able to diagnose problems, open `Preferences: LSP Settings` from the Command Palette and set the following options:
+
+| Option                  | Description                                                          |
+| ----------------------- | -------------------------------------------------------------------- |
+| `log_debug: true`       | Show verbose debug messages in the Sublime Text console.             |
+| `log_server: ["panel"]` | Log communication from and to language servers in the output panel.  |
+
+Once enabled (no restart necessary), the communication log can be seen by running `LSP: Toggle Log Panel` from the Command Palette. It might be a good idea to restart the Sublime Text and reproduce the issue again, so that the logs are clean.
```

Comment:
```suggestion
Once enabled (no restart necessary), the communication log can be seen by running `LSP: Toggle Log Panel` from the Command Palette. It might be a good idea to restart Sublime Text and reproduce the issue again, so that the logs are clean.
```